File: EJERCICIOS/PythonComoCalcu/ej1.py
''' 1. Dado dos números enteros imprimir por pantalla el producto de los mismos. 
Si el producto es más grande que 10000, imprimir su suma '''

entero1 = int(input("Ingrese un entero: \n"))
entero2 = int(input("Ingrese otro entero: \n"))
resultado = entero2 * entero1

if (resultado > 10000):
    print("La suma de los enteros ingresado es: ",entero1 + entero2)
else:
    print("El producto de los enteros ingresado es: ", resultado)

File: EJERCICIOS/Compiladores/Ej1/ej1.c
/*rear un código fuente en C que sume dos números enteros recibidos porteclado e imprima el resultado por pantalla.Luego, 
sobre el ejemplo anterior:Obtener el archivo luego de la ejecución del pre-procesador. ¿Quédirectivas son eliminadas o reemplazas?
A partir del item anterior, obtener el código assembler utilizando GCC. 
¿Para qué arquitecutura es el código?Ejecutar el ensamblado del apartado anteriorEnlazar el objeto y generar el ejecutable*/

#include <stdio.h>

int main(){

    int num1, num2, suma;

    printf("Ingrese un numero: ");
    scanf("%d", &num1);
    
    printf("Ingrese otro numero: ");
    scanf("%d", &num2);
    
    suma = num1 + num2;
    
    printf("El resultado fue: %d", suma);
    
    return 0;

}